README.md

```diff
@@ -5,3 +5,61 @@
 # Coding standards
 
 # Code reviews
+
+---
+### Pair programming
+
+**What's pair programming ?**
+
+Pair programming is a development method where two developers code on the same computer. The first, the one who writes the code is called the driver and the second observer assists the driver by correcting potential errors and proposing development alternatives.
```

Comment:
Maybe "code" should be replaced by another word like "work" to avoid confusion

Reply:
Maybe add a comma and "the" after the "second" word to ease the read

Reply:
I will replace "code" by "work" but i don't understand what do you say with "the" and "second"

Reply:
I meant that to ease the read, "and the second observer..." could be replaced by something like "and the second, the observer, assists ..."

Reply:
ok i change it